packages/manager/apps/dedicated/client/app/account/user/newAccountForm/translations/Messages_fr_FR.json

```diff
@@ -13,6 +13,7 @@
   "signup_field_legalform": "Type de compte",
   "signup_field_country": "Pays",
   "signup_field_language": "Langue",
+  "signup_field_managerLanguage": "Langue de l'interface d'administration",
```

Comment:
```suggestion
  "signup_field_managerLanguage": "Langue de l'espace client",
```
maybe
?

Reply:
i just copy pasted the suggested sentence but i think you're right it sounds better i'm going to change it